spec/system/avo/select_all_spec.rb

```diff
@@ -10,95 +10,142 @@
   let!(:salmon) { create_list :fish, spec_salmon_number, name: "Spec Salmon" }
   let!(:url) { "/admin/resources/fish?per_page=#{per_page}" }
 
-  describe "without applyed filters" do
-    context "select first page" do
-      it "releases the fish from the selected page" do
-        visit url
+  # describe "without applyed filters" do
```

Comment:
Why did we comment these tests?

Reply:
My bad, I commeted on dev when working on it and forgot to uncomment 🤦🏼